install.sh

```diff
@@ -153,7 +172,11 @@ dep_get ()
     if ! sha256_verify "${pkg_hash}" "${pkg_name}"; then
         return 1
     fi
-    tar -xzf "${pkg_name}" -C ../
+    if [[ $1 == *zip ]]; then
+        unzip "${pkg_name}" -d ../
```

Comment:
Can we assume unzip will be always present and should not add it to the dependency list?

Reply:
No, good point, I don't think we can.

Reply:
agreed as I lacked `unzip` as well: https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1000#issuecomment-962423571

Reply:
Done in  https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1000/commits/7a5e4c3369966dda121ac01a09dd999071c8dc4a (but the addition to the Darwin deps is not of course tested yet)